public/editor/stylesheets/editor.less

```diff
@@ -258,6 +260,7 @@ body {
     width: 15px;
     position: relative;
     top: 0;
+	.transition;
```

Comment:
Please keep spacing / indentation consistent. I think you're using Tabs while the source file is using spaces.